gson/src/main/java/com/google/gson/Strictness.java

```diff
@@ -0,0 +1,20 @@
+package com.google.gson;
+
+/**
+ * Modes that indicate how strictly a JSON {@linkplain com.google.gson.stream.JsonReader reader} or
+ * {@linkplain com.google.gson.stream.JsonWriter writer} follows the syntax laid out in the
+ * <a href="http://www.ietf.org/rfc/rfc4627.txt">RFC 4627 JSON specification</a>.
```

Comment:
I think this should be RFC 8259. (Throughout the PR.)

Reply:
I think I updated this in all places in the PR.